### PR TITLE
239 generative ai guidance

### LINKED ISSUE #239: Generative AI guidance
**Is your content or feature request related to a problem? Please describe.**
We would like a new section that provides guidance on using generative AI tools

### PR DESCRIPTION
**Pull request by ChrisSoderberg-ONS**
First draft of a guidance page on using generative AI. Currently located under 'resources' in the TOC

### CODE REVIEW

File: book/generative_ai.md [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+
+### Conclusion - Centaurs and reverse-centaurs
+
+Author and journalist Cory Doctorow uses the idea of [centaurs and reverse-centaurs](https://www.theguardian.com/us-news/ng-interactive/2026/jan/18/tech-ai-bubble-burst-reverse-centaur) when discussing AI-assisted work. A centaur is a human that is assisted in their work by a machine. It's nice to be a centaur, as the machine takes some of the drudgery out of work by doing the menial time-consuming tasks at a fraction of the effort, allowing you to be more productive and creative. A reverse-centaur is a human that assists a machine in their work. It's much less nice to be a reverse-centaur, as you are stripped of agency in your work - the machine dictates what you do and when you do it. When we use AI tools at work, we want to be centaurs and we don't want to be reverse-centaurs.
+
+Let's look at the following example. Say I'm working on a data pipeline, and I need a function that flags outliers in my data. I'm going to use CoPilot as my LLM-powered chatbot assistant. I give it the following prompt:
```
Comment (sarahcollyer) [inline book/generative_ai.md]: Love this example! A really small thing; can we change this section to say "you" instead of "I" please? The rest of the book uses active voice so we should keep things consistent.

File: book/generative_ai.md [hunk 2]
```diff
@@ -0,0 +1,487 @@
+# Generative AI
+
```
Comment (sarahcollyer) [inline book/generative_ai.md:2]: Can we add a draft notice up here please? Or something to encourage users to give feedback to us.

**Review by sarahcollyer — CHANGES_REQUESTED** [review]
This is absolutely fantastic! Just one relatively minor change to what you've written, and a couple of extra tasks to do and then I'm happy for this to be deployed.

Wasn't sure where to put this on the review, but I think we should also link to this chapter from the AI coding assistants section in Code Documentation.

Really great work!

File: book/generative_ai.md
Comment (zarbspace) [inline book/generative_ai.md:1]: Great work - really informative and thought provoking. 

The only minor suggestion I have is that where you use hyphens to add a point you instead break into two or three shorter sentences to avoid very long ones, and break up the long paragraphs a bit with spacing.  

E.g. instead of 

"From this point onwards, the neural network is trained to find the weights and biases that are needed to reproduce the desired outputs, which is done by gradient descent - loss is calculated, the weights and biases are modified in the direction that reduces loss, and the process repeats until the model converges (see the drop down section above for a primer on loss and gradient descent)."

replace with : 

"From this point onwards, the neural network is trained to find the weights and biases that are needed to reproduce the desired outputs. This is done using a method called gradient descent. Loss is calculated, the weights and biases are modified in the direction that reduces loss, and the process repeats until the model converges. 

The drop down section above contains a primer on loss and gradient descent."

Similarly I would embed links directly in the flow of the text if you can - see the last link in this section below. Rather than "can be found [here](link)" just link directly in the text?

The explanation for why LLMs are so ubiquitous now (given previous language models got far less attention) is a specific type of neural network architecture called a 'transformer', [invented in 2017 at Google](https://research.google/pubs/attention-is-all-you-need/) for the purpose of machine translation. This architecture represented a breakthrough in language models, as it opened up the models to '[hyperscaling](https://en.wikipedia.org/wiki/Hyperscale_computing)'. Increasing the size of the neural network and increasing the size of the training data led to predictable increases in performance, so both have increased exponentially ever since. Further discussion of the transformer architecture is beyond the scope of this guide, but [the Financial Times' Visual Storytelling Team have written an excellent introduction](https://ig.ft.com/generative-ai/).

**Review by zarbspace — APPROVED** [review]
Excellent addition, no major comments, just some minor points about style but I'm happy for merge. Great work!

**Comment by ChrisSoderberg-ONS** [comment]
Added suggested changes, re-requesting review

**Review by sarahcollyer — APPROVED** [review]
Looks amazing! Happy for it to be merged
